services/web/server/src/simcore_service_webserver/projects/_crud_handlers.py

```diff
@@ -233,17 +237,29 @@ async def list_projects(request: web.Request):
 @_handle_projects_exceptions
 async def list_projects_full_search(request: web.Request):
     req_ctx = RequestContext.parse_obj(request)
-    query_params: ProjectListFullSearchParams = parse_request_query_parameters_as(
-        ProjectListFullSearchParams, request
+    query_params: ProjectListFullSearchWithJsonStrParams = (
+        parse_request_query_parameters_as(
+            ProjectListFullSearchWithJsonStrParams, request
+        )
     )
+    if query_params.tag_ids:
+        try:
+            tag_ids_list = list(map(int, query_params.tag_ids.split(",")))
```

Comment:
IMO this is better encapsulated if you implement it as a helper member of `ProjectListFullSearchWithJsonStrParams` your model. 

The idea is that `ProjectListFullSearchWithJsonStrParams` encapsulates the pre-/post-processing, parsing and export  of the request query parameters. The model guarantees the validaty of the parameters and all the possible export formats in one place.